spec/new_relic_spec.rb

```diff
@@ -26,12 +38,30 @@
   end
 
 
+  context 'with correct setup' do
+    include_examples 'loads'
+  end
+
   context 'when NEW_RELIC_LICENSE_KEY is missing' do
     before do
       app_helper.comment_lines app_path.join('.env'), /NEW_RELIC_LICENSE_KEY/
     end
   
-    include_examples 'abort early', /NEW_RELIC_LICENSE_KEY is required/
+    context 'in the test environment' do
+      let(:app_env) { 'test' }
+
+      include_examples 'does not load'
```

Comment:
Why doesn't it load in test but it does in development? The code looks to me like they should behave the same? Also it feels like it would make sense if they behaved the same.